docs_source_files/content/getting_started_with_webdriver/locating_elements.fr.md

```diff
@@ -63,8 +63,8 @@ IWebElement fromage = driver.FindElement(By.Id("fromage"));
 IWebElement cheddar = fromage.FindElement(By.Id("cheddar"));
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-fromage = driver.find_element(id: "fromage")
-cheddar = fromage.find_elements(id: "cheddar")
+fromage = driver.find_element(id: 'fromage')
+cheddar = cheese.find_element(id: 'cheddar')
```

Comment:
hi @Madh93 ,

This code sample is different from other code sample and you have to pass 'formage' instead of cheese in the second step. Can you please change it

Reply:
Updated the code to pass `fromage` instead of `cheese`. But, what is the difference you have found?

Reply:
Cheese === fromage in french ? haha :) no issues then. 

Merci !

Reply:
Your welcome! :smile: